.github/workflows/publish_container.yml

```diff
@@ -21,6 +21,9 @@ jobs:
         run: |
           bin/garden-version "${{ inputs.version }}" | tee VERSION
           git update-index --assume-unchanged VERSION
+      - name: get current garden linux version
+        run: |
+          echo "gl_version=$(cat VERSION)" | tee -a "$GITHUB_ENV"
```

Comment:
maybe we should use `./get_version` instead.... won't make  a difference with the set version step above, but if that step ever changes using `./get_version` should be more stable